Library/CMakeLists.txt

```diff
@@ -152,6 +153,11 @@ if(WINDOWS_STORE)
     target_link_libraries(Library PRIVATE arcana)
 endif()
 
+if(APPLE)
```

Comment:
nit: What's the difference between checking APPLE, as you do here, versus checking BABYLON_NATIVE_PLATFORM as is done on line 28? I don't necessarily think changing this has to be done as part of this PR, especially since there are a lot of similar things in other places throughout our CMake code. (I'll open an issue so we can discuss, establish a pattern, and eventually propagate that everywhere.) But if there's a behavioral significance to the APPLE variable versus BGFX_NATIVE_PLATFORM, can you note it in a comment here?

Reply:
https://github.com/BabylonJS/BabylonNative/issues/55

Reply:
Nice catch! I'll change line 28 in my next PR as I'm doing some IOS stuff now.